contract/src/start-game1-proposal.js

```diff
@@ -1,7 +1,8 @@
 // @ts-check
-import { E } from '@endo/far';
+import { E } from '@endo/eventual-send';
 import { makeMarshal } from '@endo/marshal';
 import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import '@agoric/zoe/exported.js';
```

Comment:
These bloat bundles. Ugh. Please add an `XXX` comment referring to...

 - https://github.com/Agoric/agoric-sdk/issues/6512

**important**